src/problem_bank_scripts/problem_bank_scripts.py

```diff
@@ -1230,3 +1230,10 @@ def backticks_to_code_tags(html):
     html = html.replace("\\`", "`")  # Replace escaped backticks
     return html
 
+def validate_header(header_dict):
+    url = "https://raw.githubusercontent.com/open-resources/learning_outcomes/main/Masterlist.csv"
+    learning_outcomes = pd.read_csv(url)
+    # check if header is sanctioned
+    topic_list = learning_outcomes["Topic"].unique().tolist()
+    if header_dict["topic"] not in topic_list:
+        raise Exception(f"the topic '{header_dict['topic']}' is not listed in the learning outcomes")
```

Comment:
```suggestion
        raise ValueError(f"topic '{header_dict['topic']}' is not listed in the learning outcomes")
        
```
A bit more specific in the error type, If people actually read the error chain this will be more informative as the type than generic "Exception"

Reply:
very true, thank you!